packages/editor/src/client/atoms/dateTimePicker.tsx

```diff
@@ -19,18 +19,20 @@ export interface DateTimePickerProps {
 
   dateRanges?: DateTimePreset[]
   timeRanges?: DateTimePreset[]
+  disabled?: boolean
 }
 
 export function DateTimePicker({
   dateTime,
   label,
   changeDate,
   dateRanges,
-  timeRanges
+  timeRanges,
+  disabled
 }: DateTimePickerProps) {
   const {t} = useTranslation()
 
-  const [dateSelection, setDateSelection] = useState<any>(dateTime ?? new Date())
+  const [dateSelection, setDateSelection] = useState<any>(dateTime)
```

Comment:
@camilleory did you check everywhere that this change doesn't lead to unwanted effects?

Reply:
Yes, it's the only place where this customised time date picker is used, in the subscriptions, we use the usual one